docs/io-use.md

```diff
@@ -466,6 +466,14 @@ For the latest and complete information, see [Pulsar admin docs](pathname:///ref
 </Tabs>
 ````
 
+## Run a Pulsar Function before a sink connector
+
+You have the possibility to run a [Pulsar Function](functions-overview.md) in-memory before a sink connector.
+This provides better latency, less I/O and disk consumption than going through an intermediate topic.
```

Comment:
```suggestion
Running a [Pulsar Function](functions-overview.md) in memory before a sink connector provides lower latency, less I/O, and disk consumption than going through an intermediate topic.
```
I incorporated the first sentence into this one so it can be removed.